app-i18n/imhangul/imhangul-2.1.1.ebuild

```diff
@@ -12,13 +12,12 @@ SRC_URI="https://storage.googleapis.com/google-code-archive-downloads/v2/code.go
 LICENSE="LGPL-2.1"
```

Comment:
should be `LGPL-2.1+`, source say:
```
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Library General Public
 * License as published by the Free Software Foundation; either
 * version 2 of the License, or (at your option) any later version.
```

Probably the same applies to the version 3.1.1 (didn't check)